assets/js/topbutton.js

```diff
@@ -0,0 +1,20 @@
+// Scroll to top button
+(function ($) {
+    $(document).ready(function () {
+        $(window).scroll(function () {
+          if ($(this).scrollTop() > 2500) {
+            $('.top-button').fadeIn();
```

Comment:
Ha I didn't realize we used jQuery on the site. Could you please use double-quotes for these selector arguments? I see a mix of single/double-quotes in here.

Reply:
I don't know if there is a plan to get rid of jQuery at some point, but most of what you have here could be done natively with plain JS as well these days including the part where you are [animating](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) to the top.

Reply:
yea, it could be done with plain js as well. would have prefered to not have needed any JS at all, tried to see if there was a pure CSS way to do this, but I couldn't find one.

Reply:
We are moving away from jQuery, yeah, to Stencil.js. ([Here's a README](https://github.com/pulumi/docs/tree/master/components) -- it's like React + Angular => Web Components, with TypeScript. It's good stuff. 😄 )

I do think this could be simplified a bit by just using `a href="#"` (boring as that may be); animating the scroll is okay too, but since we don't really have anything else using it yet, and we're not animating the anchor-js clicks either, I'm kind of 🤷‍♂️ on it. 

Placement is tricky too, and hard to gauge without being able to interact with it (the Pulumify preview doesn't seem to be working, so I tried adding a DOM node manually with a `top-button` class, and I can see the box, but without the label, and it's floating mid-column, probably because we're on different-sized monitors):

![image](https://user-images.githubusercontent.com/274700/80662216-9414f780-8a45-11ea-8650-17b3461d047e.png)

It also gets complicated on pages with right-hand navs that require scrolling (there are many of these, particularly in the SDK docs). I suspect we could maybe tuck something away pinned to the bottom somehow, like absolutely positioned in relation to the right-hand side, with a solid background, or just below that div so that the scrolling weren't impacted. Not quite sure, but I do think your instinct of putting it at the bottom-right makes sense.

It might also be worth considering just making the ON THIS PAGE heading clickable, and/or conditionally showing an up-arrow beside it, since that's always an option, too, and never too far away:

![Screen Recording 2020-04-29 at 6 25 19 PM mov](https://user-images.githubusercontent.com/274700/80662809-1225ce00-8a47-11ea-9e89-a167bd8497e1.gif)

Reply:
I orginally just did an href and an anchor at the top of the page, but the reason I did the jquery was more to control the showing/hiding of the icon depending on how far you scrolled down the page. If not it will be ever present even when you are scrolled to the top which looked kind of silly when I tried it. Not sure how important this is to you. 

So I went ahead and was able to get the chevron icon that Praneet was suggesting to work. Looks better IMO. The reason it wasn;t working before is because I was trying to use it as a pseudo icon in the css by just adding the unicode code to the css content property. Apparantly this only works for some font awesome icons and not all. So in order to use this icon you will need to add the element to the templates or nav-bars as follows:
```<a class="top-button fas fa-chevron-up"></a>
```
This is how it looks now:

<img width="1100" alt="Screen Shot 2020-04-30 at 10 51 11 PM" src="https://user-images.githubusercontent.com/16751381/80785643-d0278580-8b35-11ea-9910-e71649d24753.png">

Reply:
Also made "On This Page" clickable to scroll to the top. I think this is an improvement.  Also thanks for the info on Stenciljs. Not that I want to add more dependency on jquery, and would love to convert to something like that, but for timesake I am inclined to just roll with the jquery for now as it is already so heavily ingrained in what we have. Let me know what you think @cnunciato. I orginally just did try an href and an anchor at the top of the page, but the reason I did the jquery was more to control the showing/hiding of the icon depending on how far you scrolled down the page. If not the scroll to top button would be ever-present on the page.